app/controllers/api/base_controller.rb

```diff
@@ -22,6 +22,11 @@ class BaseController < ActionController::Base
       render_error 'standard_error', :status => :internal_server_error, :locals => { :exception => error }
     end
 
+    rescue_from NoMethodError do |error|
+      Foreman::Logging.exception("Action failed", error)
+      render_error 'custom_error', :status => :internal_server_error, :locals => { :message => _('500 Internal Server Error') }
```

Comment:
Why is this better than showing the actual error..? The response headers already show 500, and the message shows valuable info, but in this case there is no useful info (500 is already in the response headers)